### ISSUE TITLE
--audio-source not found

### ISSUE THREAD

**Issue by Last-Order** [rigaya/NVEnc#142]
As the documentation says, --caption2ass needs Caption.dll. But a `Failed to load Caption.dll.` error throws when I use --audio-source.

My command line:
```
D:\Downloads\Compressed\NVEnc_4.37\NVEncC\x86\NVEncC.exe -i "E:/test.avs" -o "E:/output.mp4" --audio-source "E:\[190227] Last Promise\01. Last Promise.flac":1?codec=aac;bitrate=320
```

**Comment by rigaya** [comment]
Thank you for reporting, fixed in [NVEnc 4.46](https://github.com/rigaya/NVEnc/releases/tag/4.46).

**Comment by Last-Order** [comment]
Thanks for your quick fix!

**Comment by Last-Order** [comment]
I can't find `--audio-source` now..
```
C:\Users\Sora>D:\Downloads\Compressed\NVEncC_4.46_x64\NVEncC64.exe -i E:/test.avs --audio-source "E:\[190227] Last Promise\01. Last Promise.flac":1?codec=aac;bitrate=320 -o E:/output.mp4
Error: Unknown option: --audio-source

Did you mean option(s) below?
  --max-bitrate
  --audio-bitrate
```

**Comment by rigaya** [comment]
Parsing the command line for --audio-source was not working fine, I have fixed it in [NVEnc 4.47](https://github.com/rigaya/NVEnc/releases/tag/4.47).
.
